tsl/src/continuous_aggs/refresh.c

```diff
@@ -18,6 +18,7 @@
 
 #include "ts_catalog/catalog.h"
 #include "ts_catalog/continuous_agg.h"
+// #include "ts_catalog/continuous_aggs_watermark.h"
 #include <dimension.h>
```

Comment:
Remove it if you don't need it.
```suggestion
#include "ts_catalog/continuous_agg.h"
#include <dimension.h>
```

Reply:
Fixed!